src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -628,11 +632,10 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.WAVPACK;
 		} else if (value.contains("mlp")) {
 			format = FormatConfiguration.MLP;
+		} else if (value.equals("openmg")) {
+			format = FormatConfiguration.ATRAC;
 		} else if (value.contains("atrac")) {
 			format = FormatConfiguration.ATRAC;
-			if (media.getCodecV() == null && !media.getContainer().equals(FormatConfiguration.RM)) {
-				media.setContainer(FormatConfiguration.ATRAC);
-			}
 		} else if (value.equals("jpeg")) {
```

Comment:
Actually you did that because you wanted separated the "pure" Atrac (Sony) from the RealMedia ones. But other formats can have Atrac as well, so "openmg" do that nicely, without RM limitation.
Sony Atrac are handled by "openmg".
I've many RM or Sony Atrac tests files if you want test. I already test that in 2 of my PRs and it has always worked fine.

Do you have some idea where it can fail ?